util/chunk/column.go

```diff
@@ -251,6 +255,11 @@ func (c *Column) resize(n, typeSize int, isNull bool) {
 	} else {
 		c.data = make([]byte, sizeData)
 	}
+	if !isNull {
+		for j := 0; j < sizeData; j += len(emptyBuf) {
+			copy(c.data[j:], emptyBuf)
```

Comment:
Will `c.data[last_j:sizeData]` not filled if `sizeData % len(emptyBuf)` != 0

Reply:
No, it won't.